app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_serializer.py

```diff
@@ -596,7 +596,7 @@ def test_status_with_invalid_target_status_next(self):
         )
 
     def test_status_restriction(self):
-        experiment = NimbusExperimentFactory(status=NimbusExperiment.Status.LIVE)
+        experiment = NimbusExperimentFactory(status=NimbusExperiment.Status.COMPLETE)
```

Comment:
This should probably be parameterized for all the statuses that don't allow updates.  Might as well do it now while we're futzing with these things.

Reply:
Oh on second thought re: below, we can be exhaustive in the mixin tests and then we only need a single test in the serializer to test integration.